spec/support/warning.rb

```diff
@@ -57,7 +57,7 @@ def add(warning)
     end
 
     warnings  = Pathname.new(__dir__).join('warnings.yml').freeze
-    whitelist = IceNine.deep_freeze(YAML.load(warnings.read))
```

Comment:
It will cause problems with symlinks support and other yaml stuff, in this case we are loading yaml from the trusted source, so load is good enough